etna/core/mixins.py

```diff
@@ -89,6 +90,44 @@ def to_dict(self):
         params["_target_"] = BaseMixin._get_target_from_class(self)
         return params
 
+    def set_params(self, **params: dict) -> "BaseMixin":
+        """Return copy of this estimator with given parameters set.
```

Comment:
May be better to say "with modified parameters" because we keep other parameters with old values.